lambda-function/src/utils/measureConversionUtils.ts

```diff
@@ -66,15 +74,23 @@ const convertPopulations = (matPopulations: any) => {
   const populations = matPopulations.reduce((populationMap: MadiePopulationType, population: any) => {
```

Comment:
I think this function is operating on single population all the time now. Can we rename is to `convertPopulation`? also do you think `reduce` is not required anymore?

Reply:
@adongare I was wondering about the use of reduce? Unless you need both previous population and current population?